rllib/BUILD

```diff
@@ -3556,6 +3593,15 @@ py_test(
     args = ["--framework=torch", "--env=markov_soccer", "--win-rate-threshold=0.6", "--stop-iters=2", "--num-episodes-human-play=0"]
 )
 
+py_test(
+    name = "examples/self_play_league_based_with_open_spiel_markov_soccer_torch_w_rlm",
```

Comment:
"rlm" -> "rl_module"

Also: see my comment below: Can we use a command line arg here (`--use-rl-module`)?

Reply:
These will show up in CI under RL Module tests, It will be super clear in the CI UI. I'd like to keep it concise here. Also, it's a transitional solution. At some point it will become the default and we have to get rid of these stuff anyways.